source/a_release_25062019.rst

```diff
@@ -168,31 +169,31 @@ It is now possible to click on maps and receive Feature Info from WMS-layers and
 * a bug that continuously queried temporal rasters (reference: PROJ-855)
 
 Lizard Portal (demo.lizard.net)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------
 
 **Value on hoover**
 
 We have added value-on-hoover functionality to the graphs that are shown in the omnibox. This feature increases the readability of graphs. This also works for graphs generated by the line tool: |linetool| 
 
-.. |linetool| image:: /images/lineSelection.png
+.. |linetool| image:: /images/a_release_25062019_03.png
```

Comment:
Waarom nummertjes i.p.v. een beschrijvende naam?

En: gebruik yyyymmdd, dat sorteert beter.

Reply:
Dat was vooral voor mij, om overzicht te houden op welke plaatjes bij welke pagina horen.

De plaatjes voor die release notes gaan met een volgende release weg, omdat we de release notes anders willen weergeven.